### PR TITLE
Add translation keys for Pats page

### PR DESCRIPTION
**Pull request by Mysterious-Dev**
All is in the title ^^.

Related to https://github.com/modrinth/knossos/issues/1454

### CODE REVIEW

File: pages/settings/pats.vue [hunk 1]
```diff
@@ -2,25 +2,33 @@
...
+      :header="
+        editPatIndex !== null
+          ? formatMessage(messages.editPatModalTitle)
+          : formatMessage(messages.createPatModalTitle)
+      "
     >
       <div class="universal-modal">
-        <label for="pat-name"><span class="label__title">Name</span> </label>
+        <label for="pat-name">
+          <span class="label__title">{{ formatMessage(commonMessages.nameLabel) }}</span>
```
Comment (brawaru) [inline pages/settings/pats.vue:20]: This probably shouldn't be a common message, at least not for now. ‘Name’ can mean different things depending on the context.

**Review by brawaru — CHANGES_REQUESTED** [review]
One issue, but otherwise good to merge

File: pages/settings/pats.vue (hunk 1 above)
Comment (Mysterious-Dev) [reply pages/settings/pats.vue]: Fixed ^^.

**Comment by brawaru** [comment]
Sent a PR your way https://github.com/Mysterious-Dev/knossos/pull/10

**Comment by Mysterious-Dev** [comment]
> Sent a PR your way [Mysterious-Dev#10](https://github.com/Mysterious-Dev/knossos/pull/10)

It's merged ^^.

**Review by brawaru — APPROVED** [review]

**Comment by Mysterious-Dev** [comment]
Merge conflicts resolved ^^.

**Pull request merged**